Makefile

```
@@ -88,7 +88,7 @@ $(CORE)/%.o: $(CORE)/%.c
 
 .PHONY: format
 format:
-	clang-format -style=file:.clang-format -i **/*.c **/*.h
+	find . -type f -name "*.[ch]" | xargs clang-format -style=file:.clang-format -i
```

```suggestion
	find src bin tests -type f -name "*.[ch]" | xargs clang-format -style=file:.clang-format -i
```

限制下目录，因为我把构建好的 raylib 也放这了。